### PR TITLE
Use mediawiki_selenium 1.5.x

### PR DESCRIPTION
**Pull request by tobijat**
Version 1.6.1 has issues: see https://phabricator.wikimedia.org/T114368

### CODE REVIEW

**Comment by tobijat** [comment]
@WMDE-Fisch

File: Gemfile [hunk 1]
```diff
@@ -1,11 +1,11 @@
 # ruby=ruby-2.1.1
 # ruby-gemset=Wikibase
 
-source 'https://rubygems.org'
+source 'http://rubygems.org'
```
Comment (zeljkofilipin) [inline Gemfile]: Why the switch from https to http?

**Comment by zeljkofilipin** [comment]
:+1: in general, with one minor comment.

File: Gemfile (hunk 1 above)
Comment (thiemowmde) [inline Gemfile]: Is this something from @tobijat's local settings?

**Pull request merged**